File: java_work/ex0701/src/ex0701/ex03.java
package ex0701;
/*
 * 자연수 1부터 시작해서 모든 홀수를 더해 나간다. 그리고 그 합이 언제 1000을 넘어서는지
 * 그리고 1000을 넘어선 값은 얼마가 되는지 걔산하여 출력하는 프로그램을 작성
 */
public class ex03 {
	public static void main(String[] args) {
		
		int num = 1;
		int sum=0;
		while(true) {
			sum += num;
			num+=2;
			System.out.println("num = "+num);
			if(sum >1000)
				break;
		}
		System.out.println("sum = "+sum);
		
		
	}

}

File: python_work/220705/ex03.py
def main():
    print('main 시작')
    a = input("숫자를 입력하세요")
    print(type(a))
    num = int(a)
    print(type(num))
    num=int(a)
    if num > 0 :
        print(num,"은 0보다 큽니다.")

    else:
        print(num, "은 0보다 작습니다.")

    print('main 끝')


main()
